shared/entity/type.go

```diff
@@ -18,6 +18,18 @@ type typeInfo interface {
 
 	// path returns the API path for the resource. The pathPlaceholder constant should be used in place of mux variables.
 	path() []string
+
+	// APIMetricsURLPrefix defines the endpoint URL prefixes related to that type.
+	// This is used to categorize endpoints for the API rates metrics using entity types.
+	// If a type is not relevant for the metrics, this will return an empty slice.
+	apiMetricsURLPrefixes() []string
```

Comment:
the field is not exported, so the `APIMetricsURLPrefix` part of the comment isnt right

Reply:
commit message has type "stablishing"